src/lib/shadow-shim-helper-rs/src/shim_event.rs

```diff
@@ -3,6 +3,17 @@ use vasi::VirtualAddressSpaceIndependent;
 
 use crate::syscall_types::{ForeignPtr, SyscallArgs, SyscallReg, UntypedForeignPtr};
 
+#[derive(Copy, Clone, Debug, VirtualAddressSpaceIndependent)]
+#[repr(C)]
+/// Data for [`ShimEventToShim::Syscall`] and [`ShimEventToShadow::Syscall`]
+pub struct ShimEventStartRes {
```

Comment:
What does "Res" stand for? I'd usually guess "result", but I'm not sure that makes sense in this context.